newdle/core/app.py

```diff
@@ -81,6 +83,34 @@ def _handle_exception(exc):
         return jsonify(error='internal_error'), 500
 
 
+def _configure_cli(app):
+    @app.cli.command('cleanup_newdles')
+    def _cleanup_newdles():
+        """Removes old newdles from the database."""
+        last_activity_cleanup_interval = timedelta(
+            days=app.config['LAST_ACTIVITY_CLEANUP_INTERVAL']
+        )
+        final_date_cleanup_interval = timedelta(
+            days=app.config['FINAL_DATE_CLEANUP_INTERVAL']
+        )
+        now = datetime.utcnow()
+        final_newdles_to_delete = Newdle.query.filter(
```

Comment:
IIRC we said that we want to require both criteria to be met to delete - so if there's a final dt we still take into account whether it's still within the lastmod interval and in that case don't delete it yet.

Reply:
Hm, I don't think I get it. Is there any reason to check the `final_dt` interval then? Because the `lastmod` interval will always be greater since setting the `final_dt` updates the `last_update` column to the current timestamp.

Reply:
the `final_dt` may be a few months in the future. see this part in my comment on the issue:

> newdles get deleted only if both criteria are met, so if i finalize a newdle taking place in 12M, it'd be deleted after 13M (with the defaeults)

Reply:
so i think the logic you want here (can be done in a single query is):

    (now - lastmod > lastmod_threshold) AND (final_dt IS NULL OR now - final_dt > final_threshold)

Reply:
Updated, thanks.